specification.swagger.yaml

```diff
@@ -597,133 +48,229 @@ paths:
+          description: A distance radius filter to be used as part of a location filter
+            along with lat and lon
           required: false
+          schema:
+            type: number
+          example: 20.20
         - name: municipality
           in: query
-          type: string
-          description: A comma-separated list of one or more municipality ids. The returned units will be located within the municipalities\' boundaries. The municipality ids are either municipality names in Finnish, or full OCD IDs of the form ocd-division/country\:fi/kunta\:helsinki (URL encoded) See the [OpenCivicData Site](http://opencivicdata.readthedocs.io/en/latest/ocdids.html) for more information
+          description: A comma-separated list of one or more municipality ids. The returned
+            units will be located within the municipalities\' boundaries. The
+            municipality ids are either municipality names in Finnish, or full
+            OCD IDs of the form ocd-division/country\:fi/kunta\:helsinki (URL
+            encoded) See the [OpenCivicData
+            Site](http://opencivicdata.readthedocs.io/en/latest/ocdids.html) for
+            more information
           required: false
+          schema:
+            type: array
+            items:
+              type: string
+          example: helsinki, espoo
         - name: city_as_department
           in: query
-          description: Retrieve units which are _either_ owned/provided by the desired cities _or_ contained within the city's geographical boundaries. The parameter value must be an UUID identifying a top-level department which is a city.
+          description: Retrieve units which are _either_ owned/provided by the desired
+            cities _or_ contained within the city's geographical boundaries. The
+            parameter value must be an UUID identifying a top-level department
+            which is a city.
           required: false
+          schema:
+            type: array
+            items:
+              type: string
+          example: 83e74666-0836-4c1d-948a-4b34a8b90301,520a4492-cb78-498b-9c82-86504de88dce
         - name: provider_type
           in: query
-          type: string
           description: A comma-separated list of unit provider types to be used as a filter
           required: false
+          schema:
+            type: array
+            items:
+              type: integer
+          example: 1,8
         - name: provider_type__not
           in: query
-          type: string
-          description: A comma-separated list of unit provider types to be used as an exclusion filter
+          description: A comma-separated list of unit provider types to be used as an
+            exclusion filter
           required: false
+          schema:
+            type: array
+            items:
+              type: integer
+          example: 2,3
```

Comment:
Your example seems to crash the backend. ;)

Reply:
there is a bug which was found with the help of this task, very cool! Will work after the merge to master.